Dockerfile

```
@@ -0,0 +1,34 @@
+FROM rocker/r-ver:4.2.0
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    libgit2-dev \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libxml2-dev \
+    libfontconfig1-dev \
+    zlib1g-dev \
+    libharfbuzz-dev \
+    libfribidi-dev \
+    libfreetype6-dev \
+    libpng-dev \
+    libtiff5-dev \
+    libgdal-dev \
+    libgeos-dev \
+    libproj-dev \
+    libudunits2-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN Rscript -e "install.packages('terra', repos='https://cloud.r-project.org')"
+
+RUN Rscript -e "install.packages('remotes', repos='https://cloud.r-project.org')"
+RUN Rscript -e "install.packages('devtools', repos='https://cloud.r-project.org', dependencies=TRUE)"
+
+WORKDIR /home/biodiversity-horizons
+
+COPY . .
```

I think if we break this up we'll be able to re-build the container much faster when making changes to just the code since the slow part is installing the dependencies:

```suggestion
COPY NAMESPACE .
COPT DESCRIPTION .
RUN Rscript -e "remotes::install_local('.', dependencies=TRUE)"
COPY R/. R/.
COPY scripts/. scripts/.
```

I haven't tested this, so you may need to tweak this a bit. Also note that we avoid copying unnecessary files. In particular `data-raw/` shouldn't be part of the container. Also any output files should be written to an external mounted volume since otherwise they'll be done once the container stops.